### PR TITLE
Implemented custom_editor filter in member/GetAll

### PR DESCRIPTION
**Pull request by ichiaohsu**
1. Supported custom_editor and active filter
2. Modified corresponding test

### CODE REVIEW

File: models/members.go [hunk 1]
```diff
@@ -53,26 +55,78 @@ var MemberAPI MemberInterface = new(memberAPI)
...
+				err := json.Unmarshal([]byte(args.Active), &tmp)
+				if err != nil {
+					fmt.Println("active ", err.Error())
+				} else {
+					for k, v := range tmp {
+						whereString = append(whereString, fmt.Sprintf("%s %s (?)", tag, operatorHelper(k)))
+						values = append(values, v)
+					}
+				}
+			}
```
Comment (hcchien) [inline models/members.go]: I think we should have the default to handle the exception here.

File: routes/member_test.go [hunk 2]
```diff
@@ -22,42 +22,77 @@ func initMemberTest() {
...
-			return sortedMockMemberDS[i].UpdatedAt.Before(sortedMockMemberDS[j].UpdatedAt)
+		sort.SliceStable(result, func(i, j int) bool {
+			return result[i].UpdatedAt.Before(result[j].UpdatedAt)
 		})
+		err = nil
 	case "-updated_at":
-		sort.SliceStable(sortedMockMemberDS, func(i, j int) bool {
-			return sortedMockMemberDS[i].UpdatedAt.After(sortedMockMemberDS[j].UpdatedAt)
+		sort.SliceStable(result, func(i, j int) bool {
+			return result[i].UpdatedAt.After(result[j].UpdatedAt)
```
Comment (hcchien) [inline routes/member_test.go]: add the default handler.

File: models/members.go (hunk 1 above)
Comment (ichiaohsu) [reply models/members.go]: I don't understand what you mean by default. Do you mean active should be set default to not -1 ?

**Comment by ichiaohsu** [comment]
Update active validation in PR #42

**Pull request merged**